python3/cloudwatch/src/lambda_function.py

```diff
@@ -56,18 +51,18 @@ def _extract_lambda_log_message(log, log_group):
             start_split = 0
 
         message_parts = str_message[start_split:].split('\t')
-        if len(message_parts) == 3:
+        size = len(message_parts)
+        if (size == 3 or size == 4):
             log['@timestamp'] = message_parts[0]
             log['requestID'] = message_parts[1]
-            log['message'] = message_parts[2]
+            log['message'] = message_parts[size - 1]
```

Comment:
If `size == 4` what do we have in `message_parts[2]`?

Reply:
The word "INFO"
<img width="811" alt="nodejs_event" src="https://user-images.githubusercontent.com/15616771/73208951-fd882000-414f-11ea-976a-dfdd35d68dad.png">